frontend/lib/norent/homepage.tsx

```diff
@@ -125,7 +125,7 @@ const LandingPageCollectiveActionList = () => (
 );
 
 export const NorentHomePage: React.FC<{}> = () => (
-  <Page title="NoRent.org" className="content">
+  <Page title="" className="content">
```

Comment:
@sraby by setting the title to the empty string, it will be just "NoRent.org", rather than "NoRent.org - NoRent.org".